addons/web/static/src/webclient/burger_menu/burger_menu.xml

```diff
@@ -4,26 +4,26 @@
 <!-- Owl Templates -->
 
 <div t-name="web.BurgerMenu" owl="1">
-    <a type="button"
-        class="o_mobile_menu_toggle dropdown-toggle o-no-caret d-md-none"
+    <button
+        class="o_mobile_menu_toggle o-no-caret d-md-none border-0"
         title="Toggle menu" aria-label="Toggle menu"
         t-on-click="_openBurger">
         <i class="oi oi-panel-right"/>
-    </a>
+    </button>
     <t t-portal="'body'">
       <Transition name="'burgerslide'" visible="state.isBurgerOpened" leaveDuration="200" t-slot-scope="transition">
-        <div class="o_burger_menu d-flex flex-column flex-nowrap position-fixed" t-att-class="transition.className" t-on-touchstart.stop="_onSwipeStart" t-on-touchend.stop="_onSwipeEnd">
-          <div class="o_burger_menu_topbar d-flex align-items-center justify-content-between"
+        <div class="o_burger_menu position-fixed top-0 bottom-0 start-100 d-flex flex-column flex-nowrap w-100 pl-5" t-att-class="transition.className" t-on-touchstart.stop="_onSwipeStart" t-on-touchend.stop="_onSwipeEnd">
```

Comment:
ml-5, if not, we will not able to close the burger menu by clicking on the backdrop because the burger menu is over it...